src/coreComponents/constitutive/ConstitutivePassThru.hpp

```diff
@@ -100,6 +100,9 @@ struct ConstitutivePassThru< SolidBase >
     ConstitutivePassThruHandler< DamageSpectral< ElasticIsotropic >,
                                  DamageVolDev< ElasticIsotropic >,
                                  Damage< ElasticIsotropic >,
+                                 DuvautLionsSolid< DruckerPrager >,
+                                 DuvautLionsSolid< DruckerPragerExtended >,
+                                 DuvautLionsSolid< ModifiedCamClay >,
```

Comment:
In the this file,  Porous Viscous models should also be included in the `ConstitutivePassThruHandler` (line 235, 316 and 339)